Vagrantfile

```diff
@@ -52,16 +52,20 @@ def vm_cpus
 end
 
 Vagrant.configure("2") do |config|
-  # always use Vagrants insecure key
+  # always use Vagrant's insecure key
   config.ssh.insert_key = false
-
   config.vm.box = "coreos-%s" % $update_channel
-  config.vm.box_version = ">= 681.2.0"
-  config.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json" % $update_channel
+  config.vm.box_version = "= 647.2.0"
```

Comment:
This does work for coreos-stable, even if you delete that version.
```console
$ vagrant box remove coreos-stable --box-version 647.2.0 && vagrant up
```
But it won't work for coreos-alpha:
```console
$ COREOS_CHANNEL=alpha vagrant up
Bringing machine 'deis-01' up with 'virtualbox' provider...
Bringing machine 'deis-02' up with 'virtualbox' provider...
Bringing machine 'deis-03' up with 'virtualbox' provider...
==> deis-01: Running triggers before up...
==> deis-01: Box 'coreos-alpha' could not be found. Attempting to find and install...
    deis-01: Box Provider: virtualbox
    deis-01: Box Version: = 647.2.0
You specified a box version constraint with a direct box file
path. Box version constraints only work with boxes from Vagrant
Cloud or a custom box host. Please remove the version constraint
and try again.
```